example/sriov-network/infiniband/sriov-network-pod1.yaml

```diff
@@ -30,5 +30,7 @@ spec:
           nvidia.com/mlnxics: "1"
```

Comment:
while we are at it, can you fix all appearances of typo : mlnxics -> mlnxnics

Reply:
done